src/applications/disability-benefits/all-claims/pages/index.js

```diff
@@ -745,6 +750,11 @@ export const secondaryUploadSourcesChoice = {
   schema: secondaryUploadSourcesChoiceSchema,
 };
 
+export const medicalCare = {
+  uiSchema: medicalCareUISchema,
+  schema: medicalCareSchema,
+};
+
 export { ancillaryFormsWizardSummary, employmentHistory };
```

Comment:
add `medicalCare` to here

Reply:
add medical care to 
export { ancillaryFormsWizardSummary, employmentHistory };